Objects/obmalloc.c

```diff
@@ -2663,9 +2683,71 @@ _PyDebugAllocatorStats(FILE *out,
     (void)printone(out, buf2, num_blocks * sizeof_block);
 }
 
+int _PyMem_init_obmalloc(PyInterpreterState *interp, _PyRuntimeState *runtime)
+{
```

Comment:
Passing `runtime` isn't necessary, since it is accessible from the interpreter state.  (Plus, I don't think you need it in this function.)

```suggestion
int _PyMem_init_obmalloc(PyInterpreterState *interp)
{
```